stabilizer/src/main/java/com/hazelcast/stabilizer/worker/WorkerCommandRequestProcessor.java

```diff
@@ -0,0 +1,338 @@
+                throw e;
+            }
+        }
+
+        private void process(final GenericCommand command) throws Throwable {
+            final String methodName = command.methodName;
+            final String testId = command.testId;
+            final String testName = "".equals(testId) ? "test" : testId;
+
+            try {
+                final TestContainer<TestContext> test = tests.get(testId);
+                if (test == null) {
+                    // we log a warning: it could be that it is a newly created machine from mama-monkey.
+                    log.warn("Failed to process command: " + command + " no test with " +
+                            "testId " + testId + " is found");
+                    return;
+                }
+
+                final Method method = test.getClass().getMethod(methodName);
+                CommandThread commandThread = new CommandThread(command, command.testId) {
+                    @Override
+                    public void doRun() throws Throwable {
+                        log.info(String.format("%s Starting %s.%s() %s", DASHES, testName, methodName, DASHES));
+
+                        try {
+                            method.invoke(test);
+                            log.info(String.format("%s Finished %s.%s() %s", DASHES, testName, methodName, DASHES));
+                        } catch (InvocationTargetException e) {
+                            log.fatal(String.format("%s Failed %s.%s() %s", DASHES, testName, methodName, DASHES));
+                            throw e.getCause();
+                        } finally {
+                            if ("localTeardown".equals(methodName)) {
+                                tests.remove(testId);
+                            }
+                        }
+                    }
+                };
+                commandThread.start();
+            } catch (Exception e) {
+                log.fatal(format("Failed to execute test.%s()", methodName), e);
+                throw e;
+            }
+        }
+
+        private void process(MessageCommand command) {
+            Message message = command.getMessage();
+            workerMessageProcessor.submit(message);
+        }
+
+        @SuppressWarnings("unused")
+        private Long process(GetOperationCountCommand command) throws Throwable {
+            long result = 0;
+
+            for (TestContainer testContainer : tests.values()) {
+                long operationCount = testContainer.getOperationCount();
+                if (operationCount > 0) {
+                    result += operationCount;
+                }
+            }
+
+            return result;
+        }
+
+        private Map<String, Result<?>> process(GetBenchmarkResultsCommand command) {
+            String testId = command.getTestId();
+            return tests.get(testId).getProbeResults();
+        }
+    }
+
+    private HazelcastInstance getHazelcastInstance() {
```

Comment:
I normally place inner classes at the end of the file. This method should be moved up.